packages/hint/src/lib/cli/analyze.ts

```diff
@@ -20,6 +20,8 @@ import cutString from '../utils/misc/cut-string';
 import * as resourceLoader from '../utils/resource-loader';
```

Comment:
@sarvaje Can you move all the changes from  `packages/hint` in a separate commit with `New: ...`? Thanks!

Reply:
done!